README.md

```diff
@@ -1,5 +1,7 @@
 # Inforz - Informatikerzeitschrift der Fachschaft Informatik an der TU Darmstadt
 
+[![Build LaTeX document](https://github.com/Rdeisenroth/inforz/actions/workflows/build.yml/badge.svg)](https://github.com/Rdeisenroth/inforz/actions/workflows/build.yml)
```

Comment:
Can you point this to the d120 inforz repo instead?

Reply:
Fixed in [001d375](https://github.com/d120/inforz/pull/1/commits/001d375e02fe932d8ac57228533fc1060bfb6f3e)